src/api/import.js

```diff
@@ -108,6 +108,15 @@ export default async (providedOptions = {}) => {
     }
   }
 
+  packageJSON.scripts = packageJSON.scripts || {};
+  d('reading current scripts object:', packageJSON.scripts);
+
+  packageJSON.scripts.start = 'electron-forge start';
+  packageJSON.scripts.package = 'electron-forge package';
+  packageJSON.scripts.make = 'electron-forge make';
```

Comment:
I don't think so, it's not really something we can get wrong

I.e. They're gonna need the start/make/package scripts and what they currently have would be wrong.